### PR TITLE
Setup travis files

### PR DESCRIPTION
**Pull request by rami-a**
Checking if this works

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+cache:
+  directories:
+    - $HOME/shared/.pub-cache
+notifications:
+  email:
+    rami-a+github@gmail.com
+
+# Only building master means that we don't run two builds for each pull request.
+branches:
+  only: [master]
```
Comment (clocksmith) [inline .travis.yml]: newline

File: travis_script.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    # Run the formatter on all the dart files to make sure everything's linted.
+    "${localSdkPath}/bin/flutter" format -n --set-exit-if-changed .
+
+    # Run the actual tests.
+    "${localSdkPath}/bin/flutter" test
+
+    popd
+done
+
+echo "-- Success --"
```
Comment (clocksmith) [inline travis_script.sh]: newline

File: .travis.yml (hunk 1 above)
Comment (rami-a) [reply .travis.yml]: Done

File: travis_script.sh (hunk 2 above)
Comment (rami-a) [reply travis_script.sh]: Done

**Pull request merged**